src/lua/debug.lua

```diff
@@ -0,0 +1,8 @@
+local debug = require('debug')
+
+local function sourcepath()
```

Comment:
I would expect a path to a file from a function named in this way. Maybe use some derivative from 'directory'?

Reply:
Added `sourcedir()` in addition to `sourcepath()`. sourcepath() now returns full path to the file